swap.js

```diff
@@ -0,0 +1,27 @@
+const isBigEndian = () => {
+  return (new Uint8Array(new Uint32Array([0x12345678]).buffer)[0] === 0x12);
+};
+
```

Comment:
```suggestion
const isBigEndian = (new Uint8Array(new Uint32Array([0x12345678]).buffer)[0] === 0x12)
```

Make isBigEndian an constant avoiding being called every swap32LE call

I would run the benchmarks again to see the difference